server/src/test/scala/za/co/absa/atum/server/api/http/EndpointsSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//package za.co.absa.atum.server.api.http
+//
+//import org.scalatest.funspec.AnyFunSpec
+//import org.scalatest.matchers.should.Matchers
```

Comment:
basically dead code for the whole module

Reply:
still todo, or?

Reply:
let's remove this file and create a new issue for Java 8 -> Java 11, and let's put this file there

Reply:
I have removed the dependency and also the test code. Created an issue for addressing tests of http endpoints.
https://github.com/AbsaOSS/atum-service/issues/152